src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -2,10 +2,15 @@
 
 import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger logger = LogManager.getLogger(AuthenticationServiceImpl.class);
+
     @Override
     public User login(String login, String password) throws AuthenticationException {
+        logger.debug("login method was called. Params: login = {}", login);
```

Comment:
The log message is informative, but it seems like the TODO comment on line 14 is redundant now. According to the checklist, all TODO comments should be removed before submitting the solution.